dfget/core/uploader/peer_server.go

```diff
@@ -457,7 +458,12 @@ func (ps *peerServer) shutdown() {
 	ps.syncTaskMap.Range(func(key, value interface{}) bool {
 		task, ok := value.(*taskConfig)
 		if ok {
-			ps.api.ServiceDown(task.superNode, task.taskID, task.cid)
+			resp, e := ps.api.ServiceDown(task.superNode, task.taskID, task.cid)
+			if e != nil {
```

Comment:
I am wondering if it is more appropriate to handle errors in the api functions? And then we can reduce redundant code for error handling. WDYT?